boefjes/boefjes/plugins/pdio_subfinder/main.py

```diff
@@ -1,15 +1,23 @@
+import logging
+import os
+
 import docker
 
 from boefjes.job_models import BoefjeMeta
 
-SUBFINDER_IMAGE = "projectdiscovery/subfinder:v2.6.6"
-
 
 def run(boefje_meta: BoefjeMeta) -> list[tuple[set, bytes | str]]:
+    """Create image from docker and run subfinder with only active domains output."""
+    subfinder_image = f"projectdiscovery/subfinder:{os.getenv('SUBFINDER_VERSION', 'v2.6.6')}"
+    rate_limit = int(os.getenv("SUBFINDER_RATE_LIMIT", "0"))
+
     client = docker.from_env()
     input_ = boefje_meta.arguments["input"]
     hostname = input_["name"]
 
-    output = client.containers.run(SUBFINDER_IMAGE, ["-silent", "-active", "-d", hostname], remove=True)
+    logging.info("Using %s with rate limit %s", subfinder_image, rate_limit)
+    output = client.containers.run(
+        subfinder_image, ["-silent", "-active", "-rate-limit", str(rate_limit), "-d", hostname], remove=True
```

Comment:
```suggestion
        subfinder_image, ["-silent", "-active", "-rate-limit", str(int(rate_limit)), "-d", hostname], remove=True
```

Reply:
This makesit impossible to inject random stuff in the cli of subfinder.

Reply:
`rate_limit` has already been converted to `int` at line 12